Eatery/Controllers/Look Ahead/LookAheadViewController.swift

```diff
@@ -181,8 +181,14 @@ class LookAheadViewController: UIViewController {
             make.edges.equalToSuperview()
         }
 
-        tableView.register(LookAheadTableViewCell.self, forCellReuseIdentifier: CellIdentifier.eatery.rawValue)
-        tableView.register(LookAheadHeaderView.self, forHeaderFooterViewReuseIdentifier: HeaderIdentifier.header.rawValue)
+        tableView.register(
+            LookAheadTableViewCell.self,
+            forCellReuseIdentifier: CellIdentifier.eatery.rawValue
+        )
```

Comment:
just curious -- is this a line length thing or splitting up arguments across lines?

Reply:
Line length: I tried not to split arguments unless it was necessary